src/rust/src/dataframe/general.rs

```diff
@@ -368,4 +368,25 @@ impl PlRDataFrame {
             .map_err(RPolarsErr::from)?
             .into())
     }
+
+    pub fn hash_rows(
+        &mut self,
+        seed: NumericScalar,
+        seed_1: NumericScalar,
+        seed_2: NumericScalar,
+        seed_3: NumericScalar,
+    ) -> Result<Sexp> {
```

Comment:
Did something like `Result<PlRSeries>` not work?

Reply:
I thought I had tried it but actually I could simplify, thanks.